### ISSUE TITLE
When is it available in the stable version of rust?

### ISSUE THREAD

**Issue by lixiaoyong12** [rust-lang/portable-simd#409]
Currently available in the nightly version. When will the stable version be available?
```[tasklist]
### Tasks
```

**Comment by lixiaoyong12** [comment]
@alex @astraw @cuviper @adamgreig

**Comment by cuviper** [comment]
I have no idea how you picked who to ping, but I'll give an answer from the release team perspective...

Once it's synced to the rust repo, like https://github.com/rust-lang/rust/pull/122905, you can see the milestone for which release it will reach. For the date, there's also a short schedule on the [forge](https://forge.rust-lang.org/).

**Comment by bjorn3** [comment]
The `std::simd` submodule is still marked as unstable and as such not usable on stable rust. There is no ETA for when it will be stabilized. It will be stabilized once it is ready.

**Comment by lixiaoyong12** [comment]
@dpaoliello  Currently available in the nightly version. When will the stable version be available?

**Comment by calebzulawski** [comment]
As @bjorn3 said, there is no ETA on when it will be available in stable.  I'm closing this in favor of the tracking issue.

**Comment by lixiaoyong12** [comment]
> As @bjorn3 said, there is no ETA on when it will be available in stable. I'm closing this in favor of the tracking issue.

Thank you. I'm looking forward to this feature.
